### ISSUE TITLE
copy extra libs lo longer works

### ISSUE THREAD

**Issue by bbaassssiiee** [hudecof/ansible_tomcat#6]
```
TASK [hudecof.tomcat : copy extra libs] ****************************************
fatal: [default]: FAILED! => {"msg": "The conditional check 'item.state == 'enabled'' failed. The error was: error while evaluating conditional (item.state == 'enabled'): 'ansible.utils.unsafe_proxy.AnsibleUnsafeText object' has no attribute 'state'\n\nThe error appears to have been in '/root/.ansible/roles/hudecof.tomcat/tasks/setup-common.yml': line 30, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"copy extra libs\"\n  ^ here\n"}
	to retry, use: --limit @/tmp/ansible/playbook.retry
```

**Comment by hudecof** [comment]
as I wrote in #5, this role needs huge revision ;(

**Comment by hudecof** [comment]
should be fixed in release/tag v0.1.1
